python/GangaLHCb/Lib/RTHandlers/RTHUtils.py

```diff
@@ -57,12 +57,6 @@ def lhcbdiracAPI_script_template():
 def is_gaudi_child(app):
     if isType(app, Gaudi):
         return True
-
-    if isType(app, TaskApplication):
-        from GangaLHCb.Lib.Applications import GaudiPythonTask, BenderTask
-        if not isType(app, GaudiPythonTask) and not isType(app, BenderTask):
```

Comment:
@drmarkwslater should comment further - Tasks didn't work in the develop branch because TaskApplication doesn't exist anymore. I didn't immediately find the other two classes either so I removed the whole check.